bundles/org.pitest.pitclipse.launch.ui/src/org/pitest/pitclipse/launch/ui/PitMutatorsTab.java

```diff
@@ -0,0 +1,380 @@
+        mutatorsTable.getTable().setEnabled(customMutatorsButton.getSelection());
+    }
+
+    protected static void addSeparator(Composite parent) {
+        Label separator = new Label(parent, SWT.SEPARATOR | SWT.HORIZONTAL);
+        GridDataFactory.fillDefaults().span(NUMBER_OF_COLUMNS, 1).applyTo(separator);
+    }
+
+    @Override
+    public void performApply(ILaunchConfigurationWorkingCopy config) {
+        config.setAttribute(INDIVIDUAL_MUTATORS, getIndividualMutators());
+        if (isBasicMutatorGroup()) {
+            config.setAttribute(MUTATORS, mutators);
+        } else {
+            config.setAttribute(MUTATORS, getIndividualMutators());
+        }
+        try {
+            PitMigrationDelegate.mapResources(config);
+        } catch (CoreException ce) {
+            setErrorMessage(ce.getStatus().getMessage());
+        }
+    }
+
+    private String getIndividualMutators() {
+        StringBuilder sb = new StringBuilder("");
+        Iterator<Object> iterator = Arrays.asList(mutatorsTable.getCheckedElements()).iterator();
+        while (iterator.hasNext()) {
+            sb.append((String) iterator.next());
+            if (iterator.hasNext()) {
+                sb.append(',');
+            }
+        }
+        return sb.toString();
+    }
+
+    @Override
+    public void setDefaults(ILaunchConfigurationWorkingCopy workingCopy) {
+        // Intentionally left empty
+    }
+
+    private void createSpacer(Composite comp) {
+        Label label = new Label(comp, SWT.NONE);
+        GridData gd = new GridData();
+        gd.horizontalSpan = NUMBER_OF_COLUMNS;
+        label.setLayoutData(gd);
+    }
+
+    /**
+     * Selects the button that conforms to the given value.
+     * @param selectedValue the selected value
+     * @param buttons       array of buttons to select one
+     * @return true, if a button was selected
+     */
+    private boolean updateSelectionOfGroup(String value) {
+        if (groupButtons == null || value == null) {
+            return false;
+        }
+        boolean found = false;
+        for (Button button : groupButtons) {
+            boolean selection = false;
+            if (((String) button.getData()).equals(value)) {
+                selection = true;
+                found = true;
+            }
+            button.setSelection(selection);
+        }
+
+        return found;
+    }
+
+    /**
+     * @return <b>true</b>, if one of the groups from PIT is selected.<br>
+     *         <b>false</b>, if individual mutators are selected.
+     */
+    private boolean isBasicMutatorGroup() {
+        return !customMutatorsButton.getSelection();
+    }
+
+    /**
+     * Only allow save, if one of the main mutator groups is selected or at least
+     * one mutator is selected inside the mutatorTable
+     */
+    @Override
+    public boolean canSave() {
+        return isBasicMutatorGroup() || (!isBasicMutatorGroup() && mutatorsTable.getCheckedElements().length > 0);
```

Comment:
Since `||` is evaluated lazily, the `!isBasicMutatorGroup()` is completely redundant, unless I miss something...